examples/crud/main.go

```diff
@@ -38,16 +38,16 @@ func main() {
 
 	// List all sobjects
 	queryParams := sdkms.ListSobjectsParams{
-		Sort: sdkms.SobjectSort{
+		Sort: &sdkms.SobjectSort{
 			ByName: &sdkms.SobjectSortByName{},
 		},
 	}
```

Comment:
It seems the `ListSobjectsResponse` type does not compitable without Mmetadata, so you need to ensure response always comes with Metadata by requiring it.
```suggestion
	withMetadata := true
	queryParams := sdkms.ListSobjectsParams{
		Sort: &sdkms.SobjectSort{
			ByName: &sdkms.SobjectSortByName{},
		},
		WithMetadata: &withMetadata,
	}
```